assets/js/components/GlobalFilters.vue

```diff
@@ -183,15 +179,35 @@ export default Vue.extend({
           }
         }
       }
+
       return filters;
     },
   },
 
+  created() {
+    const self = this;
+    this.chronologicalFilters = [];
+
+    this.makeChronologicalFiltersFromExisting();
+    this.handleEsc();
+  },
 
   methods: {
-    removeFilter(e){
-      const remove_el = e.target.getAttribute('filter')
-      this.filters[remove_el] = null;
+    makeChronologicalFiltersFromExisting() {
+      for(var i in this.filters) {
```

Comment:
ES6 please,
```js
for(const i in ...)
```
Actually, for const `k` from key, `i` makes one expect a numeric index.